[CI] Failures in DocsClientYamlTestSuiteIT.test {yaml=reference/cat/line_113}

https://gradle-enterprise.elastic.co/s/scq64g6dtjcka

There was no reproduce line in this one, but in earlier failures looking similar, e.g https://gradle-enterprise.elastic.co/s/v75nbxeirusvy from two days ago on 7.6. Unfortunately I wasn't able to reproduce with that one either
```
./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/cat/line_113}" \
  -Dtests.seed=4A91DAF216AE3A69 \
  -Dtests.security.manager=true \
  -Dtests.locale=fr \
  -Dtests.timezone=America/St_Lucia \
  -Dcompiler.java=13 \
  -Druntime.java=8
```

Failure:
```
java.lang.AssertionError: Failure at [reference/cat:2501]: field [$body] was expected to match the provided regex but didn't
Expected: health \s+status \s+index    \s+uuid                   \s+pri \s+rep \s+docs.count \s+docs.deleted \s+store.size \s+pri.store.size\s* yellow \s+open   \s+twitter  \s+.+   \s+1   \s+1       \s+1200            \s+0      \s+\d+         \s+\d+\s* green  \s+open   \s+twitter2 \s+.+   \s+1   \s+0          \s+0            \s+0        \s+\d+          \s+\d+\s*
     but: was "health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size\ngreen  open   twitter2 YsdLw8NnTZGzSys2je8VEw   1   0          0            0        230            230\nyellow open   twitter  WJOTPDu4T2uDMZYkMXSWOw   1   1       1200            0          0              0\n"
	at __randomizedtesting.SeedInfo.seed([3E289778ECCB39B2:B67CA8A24237544A]:0)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:405)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:382)
```

Another occurence https://gradle-enterprise.elastic.co/s/2wcgdiayn2yyu

Cannot reproduce locally
```
./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/cat/line_121}" \
  -Dtests.seed=4B2CE2239EB28C5A \
  -Dtests.security.manager=true \
  -Dtests.locale=ja-JP-u-ca-japanese-x-lvariant-JP \
  -Dtests.timezone=Pacific/Rarotonga \
  -Druntime.java=8
```

Sample error message is:
```
java.lang.AssertionError: Failure at [reference/cat:2501]: field [$body] was expected to match the provided regex but didn't
Expected: health \s+status \s+index    \s+uuid                   \s+pri \s+rep \s+docs.count \s+docs.deleted \s+store.size \s+pri.store.size\s* yellow \s+open   \s+twitter  \s+.+   \s+1   \s+1       \s+1200            \s+0      \s+\d+         \s+\d+\s* green  \s+open   \s+twitter2 \s+.+   \s+1   \s+0          \s+0            \s+0        \s+\d+          \s+\d+\s*
     but: was "health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size\ngreen  open   twitter2 uizXeNU1RTWZWH8sWG2IdA   1   0          0            0        208            208\nyellow open   twitter  5w1iObXnSw6iDaSSZLeJow   1   1       1200            0          0              0\n"
```

One more similar instance: https://gradle-enterprise.elastic.co/s/qy3vyaetj2ovq

```
REPRODUCE WITH: ./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/cat/line_121}" -Dtests.seed=BE393751CBCC87B4 -Dtests.security.manager=true -Dtests.locale=sr-ME -Dtests.timezone=America/Argentina/Mendoza -Druntime.java=15
```

```
  |   |  
  | org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT > test {yaml=reference/cat/line_121} FAILED |  
  | java.lang.AssertionError: Failure at [reference/cat:2501]: field [$body] was expected to match the provided regex but didn't |  
  | Expected: health \s+status \s+index    \s+uuid                   \s+pri \s+rep \s+docs.count \s+docs.deleted \s+store.size \s+pri.store.size\s* yellow \s+open   \s+twitter  \s+.+   \s+1   \s+1       \s+1200            \s+0      \s+\d+         \s+\d+\s* green  \s+open   \s+twitter2 \s+.+   \s+1   \s+0          \s+0            \s+0        \s+\d+          \s+\d+\s* |  
  | but: was "health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size\ngreen  open   twitter2 ZUkmTVQeRGKUK4r5NHFKVA   1   0          0            0        208            208\nyellow open   twitter  adflCpA0SkqltfxqTQFtLA   1   1       1200            0        171            171\n" |  
  | at __randomizedtesting.SeedInfo.seed([BE393751CBCC87B4:366D088B6530EA4C]:0) |  
  | at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:405) |  
  | at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:382) |  
  | at jdk.internal.reflect.GeneratedMethodAccessor12.invoke(Unknown Source)
```

--- a/docs/reference/cat.asciidoc
+++ b/docs/reference/cat.asciidoc
@@ -113,13 +113,13 @@ by all the shards, not number of documents).  The `/_cat/indices` API
 is ideal.  You only need to add three things to the API request:
 
 . The `bytes` query string parameter with a value of `b` to get byte-level resolution.
-. The `s` (sort) parameter with a value of `store.size:desc` to sort the output
-by shard storage in descending order.
+. The `s` (sort) parameter with a value of `index:desc` to sort the output
+by index name in descending order.
 . The `v` (verbose) parameter to include column headings in the response.
 
 [source,console]
 --------------------------------------------------
-GET /_cat/indices?bytes=b&s=store.size:desc&v=true
+GET /_cat/indices?bytes=b&s=index:desc&v=true
 --------------------------------------------------
 // TEST[setup:my_index_huge]
 // TEST[s/^/PUT my-index-000002\n{"settings": {"number_of_replicas": 0}}\n/]
@@ -129,12 +129,11 @@ The API returns the following response:
 [source,txt]
 --------------------------------------------------
 health status index            uuid                   pri rep docs.count docs.deleted store.size pri.store.size
-yellow open   my-index-000001  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0      72171         72171
 green  open   my-index-000002  nYFWZEO7TUiOjLQXBaYJpA   1   0          0            0        230          230
+yellow open   my-index-000001  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0      72171         72171
 --------------------------------------------------
 // TESTRESPONSE[s/72171|230/\\d+/]
 // TESTRESPONSE[s/u8FNjxh8Rfy_awN11oDKYQ|nYFWZEO7TUiOjLQXBaYJpA/.+/ non_json]
-// TESTRESPONSE[skip:"AwaitsFix https://github.com/elastic/elasticsearch/issues/51619"]
 
 If you want to change the <<time-units,time units>>, use `time` parameter.
 

--- a/docs/reference/cat.asciidoc
+++ b/docs/reference/cat.asciidoc
@@ -113,13 +113,13 @@ by all the shards, not number of documents).  The `/_cat/indices` API
 is ideal.  You only need to add three things to the API request:
 
 . The `bytes` query string parameter with a value of `b` to get byte-level resolution.
-. The `s` (sort) parameter with a value of `index:desc` to sort the output
-by index name in descending order.
+. The `s` (sort) parameter with a value of `store.size:desc` and a comma with `index:asc` to sort the output
+by shard storage descending order and then index name in ascending order.
 . The `v` (verbose) parameter to include column headings in the response.
 
 [source,console]
 --------------------------------------------------
-GET /_cat/indices?bytes=b&s=index:desc&v=true
+GET /_cat/indices?bytes=b&s=store.size:desc,index:asc&v=true
 --------------------------------------------------
 // TEST[setup:my_index_huge]
 // TEST[s/^/PUT my-index-000002\n{"settings": {"number_of_replicas": 0}}\n/]
@@ -129,8 +129,8 @@ The API returns the following response:
 [source,txt]
 --------------------------------------------------
 health status index            uuid                   pri rep docs.count docs.deleted store.size pri.store.size
-green  open   my-index-000002  nYFWZEO7TUiOjLQXBaYJpA   1   0          0            0        230          230
 yellow open   my-index-000001  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0      72171         72171
+green  open   my-index-000002  nYFWZEO7TUiOjLQXBaYJpA   1   0          0            0        230          230
 --------------------------------------------------
 // TESTRESPONSE[s/72171|230/\\d+/]
 // TESTRESPONSE[s/u8FNjxh8Rfy_awN11oDKYQ|nYFWZEO7TUiOjLQXBaYJpA/.+/ non_json]

